DataCreation/DataCreationMaster.R

```diff
@@ -121,3 +121,38 @@ usethis::use_data(anolisTree, overwrite = T)
 snouter <- read.table("../DataCreation/RawData/snouterdata.txt", header=T, sep="\t")
```

Comment:
Always use 'TRUE' instead of 'T' (rcmdcheck will throw an error) (but it shouldn't matter here since the file will be not included in the package)